proto/protocolapis/astria/protocol/bridge/v1alpha1/types.proto

```diff
@@ -2,9 +2,24 @@ syntax = "proto3";
 
 package astria.protocol.bridge.v1alpha1;
 
-// A response containing the last tx hash given some bridge address,
+import "astria/primitive/v1/types.proto";
+
+// A response to the `bridge/account_last_tx_hash` ABCI query
+// containing the last tx hash given some bridge address,
 // if it exists.
 message BridgeAccountLastTxHashResponse {
   uint64 height = 2;
   optional bytes tx_hash = 3;
 }
+
+// A response to the `bridge/account_info` ABCI query
+// containing the account information given some bridge address,
+// if it exists.
+message BridgeAccountInfoResponse {
+  uint64 height = 2;
```

Comment:
Why is this not indexed starting at 1? Is field number 1 reserved for something else?

Reply:
this matches the original response types which i think you wrote :p
```
// A response containing the balance of an account.
message BalanceResponse {
  uint64 height = 2;
  repeated AssetBalance balances = 3;
}

// A response containing the current nonce for an account.
message NonceResponse {
  uint64 height = 2;
  uint32 nonce = 3;
}
```

Reply:
yeah, starting later is indeed wrong. I wasn't aware that these are with us for a year already. :-S

In this PR, let them start at 1. The others we can fix in the followup breaking these request-response types out into a separate proto package.